frontend/src/generated-client/patient/base.ts

```diff
@@ -0,0 +1,86 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * ScanHub-UI
```

Comment:
header seems to be different compared to our python info headers

Reply:
suggest handling this with an issue.